### PR TITLE
Add HTMLInputElement.files

### LINKED ISSUE #19023: Reference to HTMLInputElement.files needs documentation
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture

### What specific section or headline is this issue about?

See Also section (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture#see_also)

### What information was incorrect, unhelpful, or incomplete?

Under the [See Also](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture#see_also) section, there is a list item for HTMLInputElement.files. The item doesn't currently have documentation.

### What did you expect to see?

I expected to see a link to an individual page, or a link to a relevant section of another page.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

Possible sources for documentation:
`HTMLInputElement.files` is sort of documented on the [file input page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file), especially the section about [Getting information on selected files](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#getting_information_on_selected_files). It's alsodocumented on the [Using files from web applications](https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications#getting_information_about_selected_files) reference page.

Possible temporary fix:
On the [Value section of the file input page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#value), there is already an instance of `HTMLInputElement.files` being linked to the reference page mentioned above. If it's not feasible to make a dedicated page for `HTMLIputElement.files` right now, then using the same link as the [Value section of the file input page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#value) might be a good temporary workaround.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/attributes/capture`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/attributes/capture/index.md
* Last commit: https://github.com/mdn/content/commit/9ee89fa74200ce4dad1b0f2a7c42445330de9d9d
* Document last modified: 2022-07-29T03:57:20.000Z

</details>

### PR DESCRIPTION
**Pull request by jamesaucode**
#### Summary
Added a page for HTMLInputElement.files

#### Motivation
The HTMLInputElement.files page is missing

#### Related issues
Fixes #19023

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/htmlinputelement/files/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - HTML DOM
+  - HTMLInputElement
+  - Property
+  - NeedsCompatTable
+  - Reference
+browser-compat: api.HTMLInputElement.files
+---
+{{APIRef("File and Directory Entries API")}}
+
+The **`HTMLInputElement.files`** is a property that allows you to access the {{domxref("FileList")}} selected with the `<input type="file">` element.
```
Comment (estelle) [inline files/en-us/web/api/htmlinputelement/files/index.md:17]:
```suggestion
The **`HTMLInputElement.files`** is a property that allows you to access the {{domxref("FileList")}} selected with the [`<input type="file">`](/en-US/docs/Web/HTML/Element/input/file) element.
```

Comment (estelle) [reply files/en-us/web/api/htmlinputelement/files/index.md:17]: Should we add what happens if there is no fileList (no files selected) or if the input is not of type file?

Comment (jamesaucode) [reply files/en-us/web/api/htmlinputelement/files/index.md:17]: Good point. I will add that to the doc.

Comment (estelle) [reply files/en-us/web/api/htmlinputelement/files/index.md]: I am seeing the suggested link, but don't see what happens if there is no fileList (no files selected) or if the input is not of type file.

Comment (jamesaucode) [reply files/en-us/web/api/htmlinputelement/files/index.md]: I committed your suggestion, but still working on the other part.

Comment (jamesaucode) [reply files/en-us/web/api/htmlinputelement/files/index.md]: Just added doc for if no files are selected or if the input is not of type file

**Review by estelle — APPROVED** [review]
Thank you.

**Pull request merged**